pkg/utils/pod/scheduling.go

```diff
@@ -69,6 +69,15 @@ func IsOwnedByNode(pod *v1.Pod) bool {
 	})
 }
 
+func IsPodReady(pod *v1.Pod) bool {
+	for _, condition := range pod.Status.Conditions {
+		if condition.Type != v1.PodReady {
```

Comment:
I think you may have intended to check that the `condition.Type` is the `v1.PodRedy` type and then you want to check if this is true rather than just checking if there is _any_ condition that isn't the PodReadiness condition

Reply:
updated